### PR TITLE
Add VS marketplace badges to README

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
I saw these badges on the Powershell extension for VS Code and thought they were pretty cool. Let me know what you think. If you like them - I can add them to our other extensions as well

### CODE REVIEW

**Comment by fiveisprime** [comment]
I dig that!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Azure App Service for Visual Studio Code (Preview)
 
-[![Build Status](https://travis-ci.org/Microsoft/vscode-azureappservice.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-azureappservice) [![Release Status](https://img.shields.io/github/tag/Microsoft/vscode-azureappservice.svg?label=prerelease&colorB=0e7fc0)](https://github.com/Microsoft/vscode-azureappservice/releases)
+[![Version](https://vsmarketplacebadge.apphb.com/version/ms-azuretools.vscode-azureappservice.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/ms-azuretools.vscode-azureappservice.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) [![Build Status](https://travis-ci.org/Microsoft/vscode-azureappservice.svg?branch=master)](https://travis-ci.org/Microsoft/vscode-azureappservice)
```
Comment (PrashanthCorp) [inline README.md]: Both Version and Installs seem to link to `https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice`. Question, how does the installs badge know to fetch the number of installs?

Comment (StephenWeatherford) [reply README.md]: @PrashanthCorp Look again, the URLs are different.

**Comment by StephenWeatherford** [comment]
I did it, too!

File: README.md (hunk 1 above)
Comment (EricJizbaMSFT) [reply README.md]: No @PrashanthCorp is right. They both link to the same marketplace site (The third badge is the only different one, which links to travis ci). I'm open to suggestions, but this is what the PowerShell extension did and I can't think of anywhere else they would link to.

Comment (StephenWeatherford) [reply README.md]: > How does the installs badge know to fetch the number of installs?

Sounded like he was referring to how the image links worked...

**Pull request merged**